upgrading/topics/rhsso/intro.adoc

```diff
@@ -41,3 +43,20 @@ installations can also rollback the patch and server configuration.
 A micro release only contains the artifacts that have changed. For example if Red Hat Single Sign-On 7.2.1 contains changes to
 the server and the JavaScript adapter, but not the EAP adapter, only the server and JavaScript adapter are released and require
 updating.
+
+=== Migration from Keycloak
+
+You can migrate to Red Hat Single Sign-On, the supported Red Hat product, from Keycloak, the community project. Some differences exist in the installed software, such as the following:
+
+* Red Hat Single Sign-On uses JBoss EAP in place of WildFly for the built-in application server
```

Comment:
Either remove this, or it needs more explanation. Strictly speaking it's an implementation detail.